packages/components/pie-modal/src/index.ts

```diff
@@ -163,6 +163,7 @@ export class PieModal extends RtlMixin(LitElement) implements ModalProps {
             <dialog
                 id="dialog"
                 class="c-modal"
+                data-test-id="c-modal"
```

Comment:
The `c-` prefix is more of a CSS thing. I would change this just to `modal` or `pie-modal`.